internal/semantic/types.go

```diff
@@ -1,5 +1,7 @@
 package semantic
 
-import "github.com/google/osv-scanner/pkg/lockfile"
+import (
+	"github.com/google/osv-scanner/pkg/models"
+)
 
-type Ecosystem = lockfile.Ecosystem
+type Ecosystem = models.Ecosystem
```

Comment:
nit: we might as well just delete this entirely and use `models.Ecosystem` directly

Reply:
Yeah, I deleted types.go.